### PR TITLE
French translation update

### PR DESCRIPTION
**Pull request by KRosUser**
Update of various French translation files for several modules.

_ After (Console only, most impacted)
image
image
image
image

### CODE REVIEW

File: dll/cpl/mmsys/lang/fr-FR.rc [hunk 1]
```diff
@@ -270,7 +270,7 @@ BEGIN
     5851 "Menu démarrer"
     IDS_REACTOS_LOGOFF "Déconnexion ReactOS"
     IDS_REACTOS_LOGON "Connexion ReactOS"
-    5854 "ReactOS Explorer"
-    IDS_BROWSE_FOR_SOUND "Search for new sounds"
-    IDS_WAVE_FILES_FILTER "Wave Files (*.wav)|*.wav|"
+    5854 "Explorer ReactOS"
```
Comment (HBelusca) [inline dll/cpl/mmsys/lang/fr-FR.rc:273]:
```suggestion
    5854 "Explorateur ReactOS"
```

File: dll/win32/netcfgx/lang/fr-FR.rc [hunk 2]
```diff
@@ -200,15 +200,15 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Advanced"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "The following properties are available for this network adapter. Click the property you want to change on the left, and then select its value on the right.", -1, 7, 7, 218, 24, WS_GROUP
-    LTEXT "&Property:", -1, 7, 37, 60, 8, WS_GROUP
+    LTEXT "Les propriétés suivantes sont disponibles pour votre carte réseau. Clickez sur la propriété que vous souhaitez changer à gauche, puis choisissez sa valeur à droite.", -1, 7, 7, 218, 24, WS_GROUP
```
Comment (HBelusca) [inline dll/win32/netcfgx/lang/fr-FR.rc:203]: "pour **votre** carte réseau", or: "pour **cette** carte réseau" ?
"Clickez" --> "Cliquez"

File: dll/win32/netcfgx/lang/fr-FR.rc [hunk 3]
```diff
@@ -200,15 +200,15 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
...
+    LTEXT "&Propriété:", -1, 7, 37, 60, 8, WS_GROUP
```
Comment (HBelusca) [inline dll/win32/netcfgx/lang/fr-FR.rc:204]:
```suggestion
    LTEXT "&Propriété :", -1, 7, 37, 60, 8, WS_GROUP
```

File: dll/win32/netcfgx/lang/fr-FR.rc [hunk 4]
```diff
@@ -200,15 +200,15 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
...
     LISTBOX IDC_PROPERTY_NAME, 7, 48, 124, 124, LBS_STANDARD | WS_TABSTOP
-    LTEXT "&Value:", -1, 151, 37, 60, 8, WS_GROUP
+    LTEXT "&Valeur:", -1, 151, 37, 60, 8, WS_GROUP
```
Comment (HBelusca) [inline dll/win32/netcfgx/lang/fr-FR.rc:206]:
```suggestion
    LTEXT "&Valeur :", -1, 151, 37, 60, 8, WS_GROUP
```

File: dll/win32/netcfgx/lang/fr-FR.rc [hunk 5]
```diff
@@ -200,15 +200,15 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
 CAPTION "Advanced"
```
Comment (HBelusca) [inline dll/win32/netcfgx/lang/fr-FR.rc:200]:
```suggestion
CAPTION "Avancé"
```

File: base/applications/sndvol32/lang/fr-FR.rc [hunk 6]
```diff
@@ -113,24 +113,24 @@ END
...
-    LTEXT "These settings can be used to make fine adjustments to your audio.", -1, 44, 7, 200, 32
-    GROUPBOX "Tone Controls", IDC_ADV_TONE_CONTROLS, 7, 33, 240, 80
-    LTEXT "These settings control how the tone of your audio sounds.", -1, 17, 48, 200, 14
-    LTEXT "&Bass:", -1, 17, 62, 50, 8
-    LTEXT "Low", IDC_ADV_BASS_LOW, 77, 62, 20, 8
-    LTEXT "High", IDC_ADV_BASS_HIGH, 182, 62, 20, 8
+    LTEXT "Ces paramètres peuvent être utilisés pour des ajustements fins de l'audio.", -1, 44, 7, 200, 32
+    GROUPBOX "Contrôles du ton", IDC_ADV_TONE_CONTROLS, 7, 33, 240, 80
+    LTEXT "Ces paramètres contrôles la tonalité de l'audio.", -1, 17, 48, 200, 14
+    LTEXT "&Basse:", -1, 17, 62, 50, 8
```
Comment (HBelusca) [inline base/applications/sndvol32/lang/fr-FR.rc:123]:
```suggestion
    LTEXT "&Basse :", -1, 17, 62, 50, 8
```

File: base/applications/sndvol32/lang/fr-FR.rc [hunk 7]
```diff
@@ -113,24 +113,24 @@ END
...
+    LTEXT "Bas", IDC_ADV_BASS_LOW, 77, 62, 20, 8
+    LTEXT "Haut", IDC_ADV_BASS_HIGH, 182, 62, 20, 8
     CONTROL "", IDC_ADV_BASS_SLIDER, "msctls_trackbar32", TBS_HORZ | WS_CHILD | WS_VISIBLE | WS_TABSTOP, 97, 62, 80, 20
-    LTEXT "&Treble:", -1, 17, 90, 50, 8
-    LTEXT "Low", IDC_ADV_TREBLE_LOW, 77, 90, 20, 8
-    LTEXT "High", IDC_ADV_TREBLE_HIGH, 182, 90, 20, 8
+    LTEXT "&Aigus:", -1, 17, 90, 50, 8
```
Comment (HBelusca) [inline base/applications/sndvol32/lang/fr-FR.rc:127]:
```suggestion
    LTEXT "&Aigus :", -1, 17, 90, 50, 8
```

File: dll/win32/shell32/lang/fr-FR.rc [hunk 8]
```diff
@@ -842,8 +842,8 @@ BEGIN
     IDS_FORMAT_COMPLETE "Formatage terminé."
 
     /* Warning format system drive dialog strings */
-    IDS_NO_FORMAT_TITLE "Cannot format this volume"
-    IDS_NO_FORMAT "This volume cannot be formatted! It contains important system files in order for ReactOS to run."
+    IDS_NO_FORMAT_TITLE "Ce volume ne peut pas être formatté"
+    IDS_NO_FORMAT "Ce volume ne peut pas être formatté. Il contient des fichiers systèmes indispensables pour ReactOS."
```
Comment (HBelusca) [inline dll/win32/shell32/lang/fr-FR.rc:846]:
```suggestion
    IDS_NO_FORMAT_TITLE "Impossible de formater ce volume"
    IDS_NO_FORMAT "Ce volume ne peut pas être formaté. Il contient des fichiers systèmes indispensables pour le fonctionnement de ReactOS."
```

File: dll/win32/syssetup/lang/fr-FR.rc [hunk 9]
```diff
@@ -178,12 +178,12 @@ BEGIN
     IDS_COMPUTERSUBTITLE "Vous devez fournir un nom et un mot de passe Administrateur pour votre ordinateur."
     IDS_LOCALETITLE "Options régionales"
     IDS_LOCALESUBTITLE "Vous pouvez modifier ReactOS pour différentes régions et langues."
-    IDS_LOCALETEXT "The Standards and formats are set to %s, and the location is set to %s."
-    IDS_LAYOUTTEXT "Your default text input language and method is: %s keyboard layout"
+    IDS_LOCALETEXT "Les standards et formats sont définis à %s, et la localisation à %s."
+    IDS_LAYOUTTEXT "Vote langugage par défaut est: agencement clavier %s"
```
Comment (HBelusca) [inline dll/win32/syssetup/lang/fr-FR.rc:182]:
```suggestion
    IDS_LAYOUTTEXT "Votre langage par défaut est : agencement clavier %s"
```

File: dll/win32/syssetup/lang/fr-FR.rc [hunk 10]
```diff
@@ -204,7 +204,7 @@ BEGIN
      *   samsrv.dll:IDS_USER_ADMINISTRATOR_NAME synchronized.
      *   Also check the IDD_COMPUTERPAGE dialog.
      */
-    IDS_ADMINISTRATOR_NAME "Administrator"
+    IDS_ADMINISTRATOR_NAME "Administrateur"
```
Comment (HBelusca) [inline dll/win32/syssetup/lang/fr-FR.rc:207]: We then need to have the samsrv resources translated in french for that.

**Review by HBelusca — CHANGES_REQUESTED** [review]

**Review by learn-more — DISMISSED** [review]
Hello,
Thanks for your interest in ReactOS!
We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations).
[Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.

To change this for future PR's, you can update it for globally (for the system):
```
git config --global user.name "Your Name"
git config --global user.email you@example.com
```

Or just for the current repository:
```
git config user.name "Your Name"
git config user.email you@example.com
```

Regards,

File: base/applications/calc/lang/fr-FR.rc [hunk 11]
```diff
@@ -599,7 +599,7 @@ BEGIN
     IDS_PRESSURE_KILOPASCALS "Kilopascals"
     IDS_PRESSURE_MM_OF_MERCURY "Millimètres de mercure"
     IDS_PRESSURE_PASCALS "Pascals"
-    IDS_PRESSURE_PSI "Pounds-force per square inch"
+    IDS_PRESSURE_PSI "Livres par pouce-carrés"
```
[the remark this answers is not in the document]
Comment (KRosUser) [reply base/applications/calc/lang/fr-FR.rc:602]: ok

File: base/applications/cmdutils/label/lang/fr-FR.rc [hunk 12]
```diff
@@ -5,16 +5,16 @@ LANGUAGE LANG_FRENCH, SUBLANG_NEUTRAL
...
-  label           Specifies the label of the drive."
+  drive:          Specifie la lettre du disque.\n\
+  label           Specifie l'étiquette du disque."
     STRING_LABEL_TEXT1 "Le nom de volume du disque %c: est %s\n"
     STRING_LABEL_TEXT2 "Le disque %c: n'a pas de nom de volume\n"
     STRING_LABEL_TEXT3 "Le numéro de serie du volume est %04X-%04X\n"
     STRING_LABEL_TEXT4 "Nom de volume (32 Caractères, ENTREE si aucun) ? "
-    STRING_LABEL_PROMPT "Delete current volume label (Y/N)? "
+    STRING_LABEL_PROMPT "Supprimer l'étiquette actuelle (Y/N)? "
     STRING_LABEL_OPTIONS "YN"
```
[the remark this answers is not in the document]
Comment (KRosUser) [reply base/applications/cmdutils/label/lang/fr-FR.rc]: Yes, see PromptYesNo() in label.c
However, not robust to a STRING_LABEL_PROMPT shorter than 2 chars (or unable to load....). I'll open a dedicated PR for that if confirmed

**Comment by KRosUser** [comment]
@learn-more : commit name should be OK now
@HBelusca : Thank for for the detailed comments. Everything should be OK now.

File: base/applications/cmdutils/label/lang/fr-FR.rc (hunk 12 above)
[the remark this answers is not in the document]
Comment (HBelusca) [reply base/applications/cmdutils/label/lang/fr-FR.rc]: That robustness (or absence of it) is also present in other tools, so if you want to play with those, you need to find and fix them as well.

[the remark this answers is not in the document]
Comment (KRosUser) [reply base/applications/cmdutils/label/lang/fr-FR.rc]: Do you think it is desirable at this stage of the project ? Or a useless burden for a negligeable gain in robustness ?

**Comment by KRosUser** [comment]
> Hello, Thanks for your interest in ReactOS! We ask our contributors [to use their full name when committing](https://github.com/reactos/reactos/blob/master/CONTRIBUTING.md#rules-and-recommendations). [Please amend your commit with your full name](https://stackoverflow.com/a/3042512), and update this PR.
> 
> To change this for future PR's, you can update it for globally (for the system):
> 
> ```
> git config --global user.name "Your Name"
> git config --global user.email you@example.com
> ```
> 
> Or just for the current repository:
> 
> ```
> git config user.name "Your Name"
> git config user.email you@example.com
> ```
> 
> Regards,

done

File: base/applications/cmdutils/label/lang/fr-FR.rc (hunk 12 above)
[the remark this answers is not in the document]
Comment (HBelusca) [reply base/applications/cmdutils/label/lang/fr-FR.rc]: I don't think it's of prime necessity. Indeed, the existing tools actually appear to work with their current multiple-choice implementation.

[the remark this answers is not in the document]
Comment (KRosUser) [reply base/applications/cmdutils/label/lang/fr-FR.rc]: OK !
I let you move forward with this PR as is then ?

**Comment by learn-more** [comment]
Please note that the first commit is still under `KRosUser`

**Comment by KRosUser** [comment]
> Please note that the first commit is still under `KRosUser`

How can I fix this ?

**Comment by HBelusca** [comment]
Technically I would press the "Squash & merge" button of the github interface. This would put everything under a single commit, and use as the commit author the name and email of the PR's creator as specified in his github account.

**Comment by KRosUser** [comment]
Done
@learn-more : Is it OK now ?

**Comment by KRosUser** [comment]
@learn-more : Is there any change still requested here ?

File: base/applications/cmdutils/label/lang/fr-FR.rc [hunk 13]
```diff
@@ -4,17 +4,16 @@ LANGUAGE LANG_FRENCH, SUBLANG_NEUTRAL
...
 BEGIN
-//    STRING_LABEL_HELP "Affiche ou change le nom de volume du disque.\n\nLABEL [disque:][nomdevolume]\n"
-    STRING_LABEL_HELP "Creates, changes or deletes the volume label of a disk.\n\n\
-LABEL [drive:][label]\n\n\
-  drive:          Specifies the drive letter of a drive.\n\
-  label           Specifies the label of the drive."
+    STRING_LABEL_HELP "Crée, modifie ou supprime le nom de volume d'un disque.\n\n\
+LABEL [disque:][nom_volume]\n\n\
+  disque:          Specifie la lettre du disque.\n\
+  nom_volume       Specifie le nom du volume."
```
Comment (HBelusca) [inline base/applications/cmdutils/label/lang/fr-FR.rc:10]:
```suggestion
  disque:          Spécifie la lettre du disque.\n\
  nom_volume       Spécifie le nom du volume."
```

File: base/applications/cmdutils/label/lang/fr-FR.rc [hunk 14]
```diff
@@ -4,17 +4,16 @@ LANGUAGE LANG_FRENCH, SUBLANG_NEUTRAL
...
     STRING_LABEL_TEXT1 "Le nom de volume du disque %c: est %s\n"
     STRING_LABEL_TEXT2 "Le disque %c: n'a pas de nom de volume\n"
-    STRING_LABEL_TEXT3 "Le numéro de serie du volume est %04X-%04X\n"
-    STRING_LABEL_TEXT4 "Nom de volume (32 Caractères, ENTREE si aucun) ? "
-    STRING_LABEL_PROMPT "Delete current volume label (Y/N)? "
-    STRING_LABEL_OPTIONS "YN"
+    STRING_LABEL_TEXT3 "Le numéro de série du volume est %04X-%04X\n"
+    STRING_LABEL_TEXT4 "Nom de volume (32 caractères, ENTREE si aucun) ? "
```
Comment (HBelusca) [inline base/applications/cmdutils/label/lang/fr-FR.rc:14]:
```suggestion
    STRING_LABEL_TEXT4 "Nom de volume (32 caractères, ENTRÉE si aucun) ? "
```

File: base/applications/cmdutils/label/lang/fr-FR.rc [hunk 15]
```diff
@@ -4,17 +4,16 @@ LANGUAGE LANG_FRENCH, SUBLANG_NEUTRAL
...
+    STRING_LABEL_PROMPT "Supprimer le nom actuel (O/N)? "
```
Comment (HBelusca) [inline base/applications/cmdutils/label/lang/fr-FR.rc:15]:
```suggestion
    STRING_LABEL_PROMPT "Supprimer le nom actuel (O/N) ? "
```

File: dll/cpl/console/lang/fr-FR.rc [hunk 16]
```diff
@@ -46,7 +46,7 @@ BEGIN
     LTEXT "&Police :", IDC_STATIC, 10, 105, 33, 10
     AUTOCHECKBOX "&Gras", IDC_CHECK_BOLD_FONTS, 56, 105, 60, 10
     LISTBOX IDC_LBOX_FONTTYPE, 10, 120, 110, 40, LBS_SORT | LBS_HASSTRINGS | WS_VSCROLL | WS_TABSTOP
-    GROUPBOX "Selected Font", IDC_GROUPBOX_FONT_NAME, 7, 156, 240, 50
+    GROUPBOX "Police sélectionnée", IDC_GROUPBOX_FONT_NAME, 7, 156, 240, 50
```
Comment (HBelusca) [inline dll/cpl/console/lang/fr-FR.rc:49]: Please check that all these labels display correctly.

File: dll/cpl/console/lang/fr-FR.rc [hunk 17]
```diff
@@ -15,11 +15,11 @@ BEGIN
     AUTORADIOBUTTON "&Fenêtre", IDC_RADIO_DISPLAY_WINDOW, 142, 20, 70, 10, WS_CHILD | WS_VISIBLE | WS_TABSTOP
     AUTORADIOBUTTON "P&lein écran", IDC_RADIO_DISPLAY_FULL, 142, 40, 70, 10, WS_CHILD | WS_VISIBLE | WS_TABSTOP
     GROUPBOX "Historique des commandes", IDC_STATIC, 7, 85, 120, 77, WS_CHILD | WS_VISIBLE | WS_GROUP
-    LTEXT "Taille du &buffer :", IDC_STATIC, 14, 101, 70, 12
+    LTEXT "Taille du &tampon :", IDC_STATIC, 14, 101, 70, 12
```
Comment (HBelusca) [inline dll/cpl/console/lang/fr-FR.rc:18]:
```suggestion
    LTEXT "&Taille du tampon :", IDC_STATIC, 14, 101, 70, 12
```

Comment (HBelusca) [reply dll/cpl/console/lang/fr-FR.rc:18]: Alternative: "&Taille de la mémoire tampon"

File: dll/cpl/console/lang/fr-FR.rc [hunk 18]
```diff
@@ -15,11 +15,11 @@ BEGIN
...
     EDITTEXT IDC_EDIT_BUFFER_SIZE, 90, 97, 30, 15, ES_RIGHT | ES_NUMBER | WS_GROUP
     CONTROL "", IDC_UPDOWN_BUFFER_SIZE, UPDOWN_CLASS, UDS_SETBUDDYINT | UDS_ALIGNRIGHT |
             UDS_AUTOBUDDY | UDS_ARROWKEYS | WS_GROUP, 119, 97, 12, 15
-    LTEXT "&Nombre de buffers :", IDC_STATIC, 14, 124, 70, 12
+    LTEXT "&Nombre de tampons :", IDC_STATIC, 14, 124, 70, 12
```
Comment (HBelusca) [inline dll/cpl/console/lang/fr-FR.rc:22]: Alternative: "&Nombre de mémoires tampon :"

File: dll/cpl/console/lang/fr-FR.rc [hunk 19]
```diff
@@ -36,7 +36,7 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_CAPTION
 CAPTION "Police"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Fenêtre de Prévisualisation :", IDC_STATIC, 7, 7, 115, 10
+    LTEXT "Fenêtre de prévisualisation :", IDC_STATIC, 7, 7, 115, 10
```
Comment (HBelusca) [inline dll/cpl/console/lang/fr-FR.rc:39]: Alternative: "Aperçu fenêtre :"

File: dll/cpl/console/lang/fr-FR.rc [hunk 20]
```diff
@@ -59,9 +59,9 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_CAPTION
 CAPTION "Disposition"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Fenêtre de Prévisualisation :", IDC_STATIC, 7, 7, 115, 10
+    LTEXT "Fenêtre de prévisualisation :", IDC_STATIC, 7, 7, 115, 10
     CONTROL "", IDC_STATIC_LAYOUT_WINDOW_PREVIEW, "WinPreview", WS_BORDER | WS_CHILD | WS_VISIBLE, 7, 20, 115, 70
-    GROUPBOX "Taille du buffer de l'écran", IDC_STATIC, 130, 12, 115, 50
+    GROUPBOX "Taille du tampon de l'écran", IDC_STATIC, 130, 12, 115, 50
```
Comment (HBelusca) [inline dll/cpl/console/lang/fr-FR.rc:64]: Alternative: "Taille de mémoire tampon écran"

Comment (KRosUser) [reply dll/cpl/console/lang/fr-FR.rc:64]: ok

File: dll/cpl/console/lang/fr-FR.rc (hunk 19 above)
Comment (KRosUser) [reply dll/cpl/console/lang/fr-FR.rc:39]: ok

File: dll/cpl/console/lang/fr-FR.rc (hunk 17 above)
Comment (KRosUser) [reply dll/cpl/console/lang/fr-FR.rc:18]: ok

File: dll/cpl/console/lang/fr-FR.rc (hunk 16 above)
Comment (KRosUser) [reply dll/cpl/console/lang/fr-FR.rc:49]: Yes. With proposed modifications I have to adapt a bit but i'll post "post modification" screenshots.

File: dll/cpl/console/lang/fr-FR.rc (hunk 18 above)
Comment (KRosUser) [reply dll/cpl/console/lang/fr-FR.rc]: ok

File: base/shell/explorer/lang/fr-FR.rc [hunk 21]
```diff
@@ -194,5 +194,5 @@ END
 STRINGTABLE
 BEGIN
     IDS_TASKBAR_STARTMENU_PROP_CAPTION "Barrre des tâches et menu démarrer"
```
Comment (HBelusca) [inline base/shell/explorer/lang/fr-FR.rc:196]:
```suggestion
    IDS_TASKBAR_STARTMENU_PROP_CAPTION "Barre des tâches et Menu Démarrer"
```

File: dll/cpl/console/lang/fr-FR.rc [hunk 22]
```diff
@@ -36,7 +36,7 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_CAPTION
 CAPTION "Police"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Fenêtre de Prévisualisation :", IDC_STATIC, 7, 7, 115, 10
+    LTEXT "Aperçu :", IDC_STATIC, 7, 7, 115, 10
```
Comment (HBelusca) [inline dll/cpl/console/lang/fr-FR.rc:39]:
```suggestion
    LTEXT "Aperçu de la fenêtre :", IDC_STATIC, 7, 7, 115, 10
```

File: dll/cpl/console/lang/fr-FR.rc [hunk 23]
```diff
@@ -59,9 +59,9 @@ STYLE DS_SHELLFONT | WS_CHILD | WS_CAPTION
 CAPTION "Disposition"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Fenêtre de Prévisualisation :", IDC_STATIC, 7, 7, 115, 10
+    LTEXT "Aperçu :", IDC_STATIC, 7, 7, 115, 10
```
Comment (HBelusca) [inline dll/cpl/console/lang/fr-FR.rc:62]:
```suggestion
    LTEXT "Aperçu de la fenêtre :", IDC_STATIC, 7, 7, 115, 10
```

File: dll/cpl/console/lang/fr-FR.rc [hunk 24]
```diff
@@ -15,11 +15,11 @@ BEGIN
     AUTORADIOBUTTON "&Fenêtre", IDC_RADIO_DISPLAY_WINDOW, 142, 20, 70, 10, WS_CHILD | WS_VISIBLE | WS_TABSTOP
     AUTORADIOBUTTON "P&lein écran", IDC_RADIO_DISPLAY_FULL, 142, 40, 70, 10, WS_CHILD | WS_VISIBLE | WS_TABSTOP
     GROUPBOX "Historique des commandes", IDC_STATIC, 7, 85, 120, 77, WS_CHILD | WS_VISIBLE | WS_GROUP
-    LTEXT "Taille du &buffer :", IDC_STATIC, 14, 101, 70, 12
+    LTEXT "&Taille de la mémoire tampon", IDC_STATIC, 14, 97, 70, 24
```
Comment (HBelusca) [inline dll/cpl/console/lang/fr-FR.rc:18]:
```suggestion
    LTEXT "&Taille de la mémoire tampon :", IDC_STATIC, 14, 97, 70, 24
```

**Review by HBelusca — APPROVED** [review]

**Pull request merged**